lib/wickedbase.pm

```diff
@@ -995,6 +1000,7 @@ sub pre_run_hook {
         set_var('WICKED_TCPDUMP_PID', script_output('echo $CHECK_TCPDUMP_PID'));
     }
     $self->upload_wicked_logs('pre');
+    $self->{pre_run_check_logs_cursor} = $self->check_logs_cursor() if ($self->{name} ne 'before_test');
```

Comment:
Instead of condition by looking in which module it is, wouldn't it be better to check if logs exists ?

Reply:
The thing is, that we do not wanna limit the logs check on the first boot. Because we would like to see also the logs during boot.
This is actually questionable but I think it makes sense, WDYT?